controllers/space/space_controller_test.go

```diff
@@ -765,15 +765,16 @@ func TestUpdateSpaceTier(t *testing.T) {
 		})
 	})
 
-	t.Run("tier update (same tier but updated references)", func(t *testing.T) {
+	t.Run("tier update for single space (same tier but updated references)", func(t *testing.T) {
 		// get an older basic tier (with outdated references) that the nstemplateset can be referenced to for setup
 		olderBasicTier := tiertest.BasicTier(t, tiertest.PreviousBasicTemplates)
 		// given that Space is set to the same tier that has been updated and the corresponding NSTemplateSet is not up-to-date
 		s := spacetest.NewSpace("oddity",
 			spacetest.WithTierNameAndHashLabelFor(olderBasicTier),
 			spacetest.WithSpecTargetCluster("member-1"),
 			spacetest.WithStatusTargetCluster("member-1"), // already provisioned on a target cluster
-			spacetest.WithFinalizer())
+			spacetest.WithFinalizer(),
+			spacetest.WithCondition(spacetest.Ready()))
 		hostClient := test.NewFakeClient(t, s, basicTier)
 		nsTmplSet := nstemplatetsettest.NewNSTemplateSet("oddity", nstemplatetsettest.WithReferencesFor(olderBasicTier), nstemplatetsettest.WithReadyCondition()) // NSTemplateSet has references to old basic tier
 		member1Client := test.NewFakeClient(t, nsTmplSet)
```

Comment:
could you please either set the `LastExecutedUpdate` to something older than 2 seconds ago explicitly or add a comment explaining that the update is possible because the `LastExecutedUpdate` is equal to zero. it's not really clear if you don't know the implementation detail.

Reply:
done in 68069de